pkg/landscaper/controllers/deployitem/reconcile_new.go

```diff
@@ -118,11 +118,12 @@ func (con *controller) isPickupTimeoutExceeded(log logging.Logger, di *lsv1alpha
 	return false, &requeue
 }
 
-func (con *controller) writePickupTimeoutExceeded(ctx context.Context, log logging.Logger, di *lsv1alpha1.DeployItem) error {
+func (con *controller) writePickupTimeoutExceeded(ctx context.Context, di *lsv1alpha1.DeployItem) error {
 	// no deployer has picked up the deploy item within the timeframe
 	// => pickup timeout
-	logger := log.WithValues("operation", "PickupTimeoutExceeded")
-	logger.Debug("pickup timeout occurred")
+	logger, ctx := logging.FromContextOrNew(ctx, nil)
+	logger = logger.WithValues(lc.KeyMethod, "writePickupTimeoutExceeded")
```

Comment:
```suggestion
	logger, ctx := logging.FromContextOrNew(ctx, nil, lc.KeyMethod, "writePickupTimeoutExceeded")
```

Reply:
Again the `KeyMethod` logging key, which is only for the current method, but should not be passed in a context to sub methods.